doc/src/site/sphinx/deployment/terraform_emr.rst

```diff
@@ -1,70 +1,69 @@
-   - ``sw_patch_version`` (optional) - Sparkling Water minor version. Defaults to SUBST_SW_MINOR_VERSION
+   - ``aws_region`` (optional) - AWS region. Defaults to ``us-east-1``.
+   - ``aws_emr_version`` (optional) - EMR version. Defaults to ``emr-2.20.0``.
+   - ``aws_core_instance_count`` (optional) - Number of worker nodes. Defaults to ``2``.
+   - ``aws_instance_type`` (optional) - type of EC2 instances. Defaults to ``m3.xlarge``.
+   - ``sw_major_version`` (optional) - Sparkling Water major version. Defaults to SUBST_SW_MAJOR_VERSION.
+   - ``sw_patch_version`` (optional) - Sparkling Water minor version. Defaults to SUBST_SW_MINOR_VERSION.
+
 
+ - ``default`` module  (**/templates/build/terraform/aws**). This module is a combination of the two previous modules. It starts the network infrastructure and starts EMR with Sparkling Water on top of it. 
 
- - ``default`` module  (``/templates/build/terraform/aws``). This module is combination of the two previous modules. It starts the network
-  infrastructure and starts EMR with Sparkling Water on top of it.
   This module accepts the following arguments:
 
    - ``aws_access_key`` (mandatory) - access key to access AWS
    - ``aws_secret_key`` (mandatory) - secret key to access AWS
-   - ``aws_region`` (optional) - AWS region. Defaults to ``us-east-1``
-   - ``aws_emr_version`` (optional) - EMR version. Defaults to ``emr-2.20.0``
-   - ``aws_core_instance_count`` (optional) - Number of worker nodes. Defaults to ``2``
-   - ``aws_instance_type`` (optional) - type of EC2 instances. Defaults to ``m3.xlarge``
-   - ``sw_major_version`` (optional) - Sparkling Water major version. Defaults to SUBST_SW_MAJOR_VERSION
-   - ``sw_patch_version`` (optional) - Sparkling Water minor version. Defaults to SUBST_SW_MINOR_VERSION
+   - ``aws_region`` (optional) - AWS region. Defaults to ``us-east-1``.
+   - ``aws_emr_version`` (optional) - EMR version. Defaults to ``emr-2.20.0``.
+   - ``aws_core_instance_count`` (optional) - Number of worker nodes. Defaults to ``2``.
+   - ``aws_instance_type`` (optional) - type of EC2 instances. Defaults to ``m3.xlarge``.
+   - ``sw_major_version`` (optional) - Sparkling Water major version. Defaults to SUBST_SW_MAJOR_VERSION.
+   - ``sw_patch_version`` (optional) - Sparkling Water minor version. Defaults to SUBST_SW_MINOR_VERSION.
 
 
-We provide these 3 templates as we realize some users have already some network infrastructure on their
-AWS clusters available and they don't want to use template which create it again.
+We provide these 3 templates as we realize some users already have some network infrastructure on their
+AWS clusters available, and they don't want to use a template that creates it again.
 
-To start any template you want, please run
+To start any template you want, please run the following in the corresponding **/templates/build/terraform/aws/modules/** directory:
```

Comment:
Here the path is not valid for all the 3 templates because the `default` one is inside `/templates/build/terraform/aws`. I would just mention here something like `in the corresponding module directory`

Reply:
Done. Just pushed a change.